packages/rancher-windows-gmsa/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default
+  name: {{ .Release.Name}}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "gmsa.chartref" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      securityContext:
+        runAsUser: 1000

```

Comment:
should be overridable and exposed to the end usr in case they want to change the user that runs this.

Example: https://github.com/rancher/charts/blob/502035b9ac9763e2ebbdde5c05f2137862bb2678/charts/rancher-logging/100.0.1%2Bup3.15.0/values.yaml#L81-L93

https://github.com/rancher/charts/blob/502035b9ac9763e2ebbdde5c05f2137862bb2678/charts/rancher-logging/100.0.1%2Bup3.15.0/templates/deployment.yaml#L48-L53

You probably only need containerSecurityContext, not podSecurityContext